docs/ABOUT.md

```diff
@@ -0,0 +1,43 @@
+---
+title: About Documentation
```

Comment:
I remember when we spoke... I was looking for a quick description of Cosmos Cash and found instead descriptions of the docs, let's see how we can abbreviate this information or move to another sub level, let's make the cosmos cash docs about the project, not the docs